### PR TITLE
Add Myanmar to Schema's

### PR DESCRIPTION
**Pull request by callumknights**
Part of work to finish the country name change from burma to myanmar.

### CODE REVIEW

File: formats/shared/definitions/_specialist_document.jsonnet [hunk 1]
```diff
@@ -589,6 +589,7 @@
             "montenegro",
             "morocco",
             "mozambique",
+	    "myanmar",
```
Comment (cbaines) [inline formats/shared/definitions/_specialist_document.jsonnet]: The indentation looks a bit off here.

File: formats/shared/definitions/_specialist_document.jsonnet [hunk 2]
```diff
@@ -1631,6 +1632,7 @@
...
```
Comment (cbaines) [inline formats/shared/definitions/_specialist_document.jsonnet]: Same here https://github.com/alphagov/govuk-content-schemas/pull/923/commits/11c95b96e364acfeb2fbccc3ea302382222d23fd#r330930781

File: formats/shared/definitions/_specialist_document.jsonnet (hunk 1 above)
Comment (callumknights) [reply formats/shared/definitions/_specialist_document.jsonnet]: Done!

File: formats/shared/definitions/_specialist_document.jsonnet (hunk 2 above)
Comment (callumknights) [reply formats/shared/definitions/_specialist_document.jsonnet]: Also done!

**Pull request merged**